tests/scalers/aws-dynamodb-streams.test.ts

```diff
@@ -0,0 +1,209 @@
+import test from 'ava'
```

Comment:
Sorry for being pain, right now, we are in the process in migrating TS based e2e tests to Go. Do you think you can rewrite this test to use Go?

You can follow this https://github.com/kedacore/keda/pull/3079, and once it is merged, you can use it as a base for this test. 

Thanks!

Reply:
@zroubalik Yes. I'm happy to rewrite the test in Golang!

Reply:
Awesome, thanks. So let's wait until the e2e tests migration PR is merged (it should be hopefully by the end of this week).

Reply:
@yokawasa FYI  he initial work on Golang based e2e tests has been done, you can try to follow the same pattern now.

Reply:
@zroubalik Thank you for the instruction! I've added Go based e2e tests for dynamodb streams scaler. Please reivew it https://github.com/kedacore/keda/pull/3121/commits/5ca8d02a97b8890580ea385c6b8d7011022f5707

Reply:
Could you remove the tests in ts? the idea is having only in Golang so doesn't make sense adding also in ts